.buildbot.sh

```diff
@@ -243,6 +243,14 @@ for b in collect_and_decode promote; do
     YKB_TRACER=hwt cargo bench --bench "${b}" -- --profile-time 1
 done
 
+# Test some BF programs.
+git clone https://github.com/ykjit/ykcbf.git
+cd ykcbf
+PATH=$(pwd)/../bin:${PATH} YK_BUILD_TYPE=debug make
```

Comment:
did you want to test release mode and serialised compilation combos? We don't have to test them all of course.